packages/playground/blueprints/src/lib/steps/define-wp-config-consts.ts

```diff
@@ -1,5 +1,5 @@
 import { StepHandler } from '.';
-import { updateFile } from './common';
+import { VFS_CONFIG_FILE_BASENAME, updateFile } from './common';
```

Comment:
Could this constant be defined in this file?

Reply:
Also, how about `VFS_TMP_DIRECTORY` as a name, since that seems to be the intent?

Reply:
done! https://github.com/WordPress/wordpress-playground/pull/415/commits/2e9b0dd352717685917d01c00ceb4b34c519f0d1